aspnetcore/blazor/components/render-modes.md

```diff
@@ -142,6 +142,34 @@ Technically, `@rendermode` is both a Razor *directive* and a Razor *directive at
 > [!NOTE]
 > Component authors should avoid coupling a component's implementation to a specific render mode. Instead, component authors should typically design components to support any render mode or hosting model. A component's implementation should avoid assumptions on where it's running (server or client) and should degrade gracefully when rendered statically. Specifying the render mode in the component definition may be needed if the component isn't instantiated directly (such as with a routable page component) or to specify a render mode for all component instances.
 
+## Apply a render mode to the entire app
+
+To set the render mode for the entire app, indicate the render mode at the highest-level interactive component in the app's component hierarchy that isn't a root component.
```

Comment:
A concrete example here will be good. It's impossible for anyone without context to understand what is being said here. (To give you an idea, I have context and I had to stop to think about it).

Reply:
The definition of a render mode and how to set it for component instances and component definitions were just explained in the preceding sections. What's not clear?

Reply:
I did not go through the previous section. Let me read through it